pkgs/by-name/ot/otb/package.nix

```
@@ -0,0 +1,113 @@
+{
+  cmake,
+  fetchFromGitHub,
+  makeWrapper,
+  lib,
+  stdenv,
+  swig,
+  which,
+  boost,
+  curl,
+  gdal,
+  itk_4_13,
+  libsvm,
+  libgeotiff,
+  muparser,
+  muparserx,
+  opencv,
+  perl,
+  python3,
+  extraPythonPackages ? ps: with ps; [],
+  shark,
+  tinyxml,
+}:
+
+let
+    pythonInputs = with python3.pkgs; [
+      numpy
+    ] ++ (extraPythonPackages python3.pkgs);
+in
+  stdenv.mkDerivation (finalAttrs: {
+    pname = "otb";
+    version = "9.0.0";
+
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "otb";
  version = "9.0.0";

```

please fix the remaining file, too

---

Thanks @SuperSandro2000 !

I have fixed all!